src/components/common/cpc-debrief-card/cpc-debrief-card.ts

```diff
@@ -52,11 +61,14 @@ export class CPCDebriefCardComponent implements OnInit {
   }
 
   isPass = (): boolean => {
-    return this.testOutcome === TestOutcome.PASS;
+    return this.testOutcome === TestOutcome.PASS || this.testOutcome === OutcomeType.Passed;
   }
 
   ngOnInit(): void {
     this.questions = [this.question1, this.question2, this.question3, this.question4, this.question5];
+    if (this.isDetailedTestView) {
+      this.cardHeader = 'Debrief';
```

Comment:
is this heading change in the ACs?

Reply:
It displays as 'Debrief' in the screenshot that Lewis attached which is what I was referencing when designing this component, but no its not specifically mentioned in Lukes' ACs, just seemed wrong to keep it as 'Test details' when it specifically says 'Debrief' in the attachment, this doesnt affect anything outside of the detailed test view

Reply:
just confirmed with Luke - this is how they want it